code/obj/flock/structure/compute.dm

```diff
@@ -1,6 +1,7 @@
 /obj/flock_structure/compute
 	name = "weird lookin' thinking thing"
 	desc = "It almost looks like a terminal of some kind."
+	flock_desc = "The remains of a human computer, adapted to provide compute power to the Flock."
```

Comment:
```suggestion
	flock_desc = "A computing node that provides compute power to the Flock."
```
Or similar? Feel like mentioning human computer isn't much necessary, for conciseness/relevancy reasons (already been converted), but either probably fine